stories/dynamic-side-content/dynamic-side-content.stories.js

```diff
@@ -0,0 +1,157 @@
+        </div>
+    </div>
+`;
+
+positionBottom.storyName = 'Below';
+positionBottom.parameters = {
+    docs: {
+        iframeHeight: 160,
+        storyDescription:
+            'To display side content below the main content area, keep `fd-dynamic-side__side` as the last child element and add the `fd-dynamic-side--below` modifier class.'
+    }
+};
+
+export const equalSplit = () => `
+    <div class="fd-dynamic-side fd-dynamic-side--equal-split">
+        <div class="fd-dynamic-side__main">
+            <div class="docs-layout-grid-bg docs-layout-grid-bg--color-9">Main</div>
+        </div>
+        <div class="fd-dynamic-side__side">
+            <div class="docs-layout-grid-bg docs-layout-grid-bg--color-6">Side</div>
+        </div>
+    </div>
+`;
+
+equalSplit.storyName = 'Equal split';
+equalSplit.parameters = {
+    docs: {
+        iframeHeight: 160,
+        storyDescription:
+            'The equal split mode provides users with a 50:50 view of the main content and side content, which enables users to show more data i.e. for comparison purposes. To enable equal split mode, add the `fd-dynamic-side--equal-split` modifier class to the container.'
+    }
+};
+
+export const responsiveness = () => `
+    <h3>On extra large screen the main part occupies 75% wide</h3>
+    <div class="fd-dynamic-side fd-dynamic-side--xl">
+        <div class="fd-dynamic-side__side">
+            <div class="docs-layout-grid-bg docs-layout-grid-bg--color-6">Side</div>
+        </div>
+        <div class="fd-dynamic-side__main">
+            <div class="docs-layout-grid-bg docs-layout-grid-bg--color-9">Main</div>
+        </div>
+    </div>
+    <h3>On middle screen the main part occupies 66.66% wide</h3>
+    <div class="fd-dynamic-side fd-dynamic-side--md" style="max-width: 920px">
+        <div class="fd-dynamic-side__side">
+            <div class="docs-layout-grid-bg docs-layout-grid-bg--color-6">Side</div>
+        </div>
+        <div class="fd-dynamic-side__main">
+            <div class="docs-layout-grid-bg docs-layout-grid-bg--color-9">Main</div>
+        </div>
+    </div>
+    <h3>On small screen the main part occupies 100% wide and the side content gets hidden</h3>
+    <div class="fd-dynamic-side fd-dynamic-side--sm" style="max-width: 420px">
+        <div class="fd-dynamic-side__side">
+            <div class="docs-layout-grid-bg docs-layout-grid-bg--color-6">Side</div>
+        </div>
+        <div class="fd-dynamic-side__main">
+            <div class="docs-layout-grid-bg docs-layout-grid-bg--color-9">Main</div>
+        </div>
+    </div>
+`;
+
+responsiveness.storyName = 'Responsiveness';
+responsiveness.parameters = {
+    docs: {
+        iframeHeight: 160,
+        storyDescription:
+            'To get the component rendered for the specific screen size use `fd-dynamic-side--xl` / `fd-dynamic-side--md` / `fd-dynamic-side--sm` modifier classes respectively.'
```

Comment:
To display the dynamic side content for specific screen sizes, use the respective modifier classes:

Size | Modifier class
:---- | :--------------
Small | `fd-dynamic-side--sm`
Medium | `fd-dynamic-side--md`
Large | `fd-dynamic-side--lg`
Extra large | `fd-dynamic-side--xl`

Reply:
Good, but we do not have `Large` version

Reply:
We can write 

Medium & Large | fd-dynamic-side--md
-- | --

Reply:
Changed